validateGOTermDescriptions.py

```diff
@@ -32,7 +32,17 @@ def validateGOTermDescriptions(pmid: str):
         
         goTermAPIInfo = json.loads(res.text)
             
-        desc: str = goTermAPIInfo["results"][0]["name"]
+        desc: str = ""
+        if goTermAPIInfo["results"]:
+            desc = goTermAPIInfo["results"][0]["name"]
+        else:
+            rejectedGOTerms.append({
+                "id": term["id"],
+                "description": term["description"],
+                "actualDescription": "No results found"
```

Comment:
All of this looks good, but I think it's worth getting rid of the `actualDescription` field here just to keep it cleaner in case we decide to process the rejected data in some way.

Reply:
No prob Noah, many thanks for advice! I go to change it.